userspace/libsinsp/test/container_info.ut.cpp

```diff
@@ -41,11 +41,11 @@ TEST(sinsp_container_lookup_test, default_values)
 	ASSERT_EQ(500, lookup.delay());
 }
 
-TEST(sinsp_container_lookup_test, custom)
+TEST_P(sinsp_container_lookup_test, delays_match)
 {
-	short max_retry = 5;
-	short max_delay_ms = 1000;
-	std::vector<short> expected_delays{125, 250, 500, 1000, 1000};
+	short max_retry = std::get<0>(GetParam());
+	short max_delay_ms = std::get<1>(GetParam());
+	std::vector<short> expected_delays = std::get<2>(GetParam());
```

Comment:
```suggestion
	short max_retry;
	short max_delay_ms;
	std::vector<short> expected_delays;
	std::tie(max_retry, max_delay_ms, expected_delays) = GetParam();
```

Reply:
yep, I think I considered this but I don't see a substantial difference.